src/sidebar/services/annotations-exporter.tsx

```diff
@@ -81,7 +81,7 @@ export class AnnotationsExporter {
         `Created at: ${formatDateTime(new Date(annotation.created))}`,
         `Author: ${extractUsername(annotation)}`,
         page ? `Page: ${page}` : undefined,
-        `Quote: "${quote(annotation)}"`,
+        !isReply(annotation) ? `Quote: "${quote(annotation)}"` : undefined,
```

Comment:
`Quote: "null"` is still produced for page notes. More generally it is possible to create annotations via the API which have selectors but not quote selectors. These will currently always orphan in the client, but that could change if we introduce non-text best ways of anchoring (eg. image annotations) in future.

Reply:
Then I guess the right approach would be to check if `quote(annotation)` is not null. If it's null, then we don't render it regardless the type of annotation.